lib/sandbox/sandbox.go

```diff
@@ -23,13 +23,13 @@ import (
 // NetNs handles data pertaining a network namespace
 type NetNs struct {
 	sync.Mutex
-	ns       ns.NetNS
+	nn       ns.NetNS
```

Comment:
I'd rather use `netNS` or some such here to reduce the risk of future collisions.

Reply:
Yes I like netNS

Reply:
alrighty